src/main/java/racingcar/step3/CarRacingGame.java

```diff
@@ -0,0 +1,15 @@
+package racingcar.step3;
+
+
+public class CarRacingGame {
+    public static void main(String[] args) {
+
+        int numberOfCars = InputView.getNumberOfCars();
+        Race race = new Race(numberOfCars);
+
+        int numberOfAttempts = InputView.getNumberOfAttempts();
+        race.start(numberOfAttempts);
```

Comment:
```suggestion
        int numberOfCars = InputView.getNumberOfCars();
        int numberOfAttempts = InputView.getNumberOfAttempts();
        
        Race race = new Race(numberOfCars);
        race.start(numberOfAttempts);
```

입력값 부터 받은 다음 로직을 전개하는 방식이 더 자연스러울 것 같아요

Reply:
```java
var dto = InputView.getNumberOfCarsAndAttempts();
```
데이터 전달 목적으로 dto 타입을 받아도 괜찮을 것 같네요
위 코드는 예시입니다.